Formula/tfproviderlint.rb

```diff
@@ -0,0 +1,38 @@
+class Tfproviderlint < Formula
+  desc "Terraform Provider Lint Tool"
+  homepage "https://github.com/bflad/tfproviderlint"
+  url "https://github.com/bflad/tfproviderlint/archive/v0.27.1.tar.gz"
+  sha256 "92bbef65ccc2a2947e5dc8e0cfdf20d0485dbf87a21fbc10c865ff25210fb6a8"
+  license "MPL-2.0"
+  head "https://github.com/bflad/tfproviderlint.git", branch: "main"
+
+  depends_on "go" => [:build, :test]
+
+  resource "test_resource" do
+    url "https://github.com/russellcardullo/terraform-provider-pingdom/archive/v1.1.3.tar.gz"
+    sha256 "3834575fd06123846245eeeeac1e815f5e949f04fa08b65c67985b27d6174106"
+  end
```

Comment:
Can't we test this without cloning a repo?

Reply:
I was trying that, but the setup is kind of complicated, that is why I used the test repo for testing the functionality.